src/app/Processor/ProcessorEdit/ProcessorEdit.tsx

```diff
@@ -408,6 +411,37 @@ const ProcessorEdit = (props: ProcessorEditProps): JSX.Element => {
                                   ouiaId={"transformation-description"}
                                 >
                                   {t("processor.addTransformationDescription")}
+                                  <Popover
+                                    headerContent={t(
+                                      "processor.transformationTemplate"
+                                    )}
+                                    bodyContent={
+                                      <Trans
+                                        i18nKey={
+                                          "openbridgeTempDictionary:processor.transformationTemplateTooltip"
+                                        }
+                                        components={[
+                                          <ExternalLink
+                                            key="qute-reference-link"
+                                            testId="qute-reference-link"
+                                            href="https://quarkus.io/guides/qute-reference"
+                                          />,
+                                        ]}
+                                      />
+                                    }
+                                  >
+                                    <button
+                                      type="button"
+                                      aria-label={t(
+                                        "processor.moreInfoForTransformationTemplate"
+                                      )}
+                                      onClick={(e): void => e.preventDefault()}
+                                      aria-describedby="form-group-label-info"
```

Comment:
Minor obs.

You should probably update the value of `aria-describedby`. Maybe add an `id` attr to the parent `Text` component and use that?

Reply:
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/121/commits/3717fe2818801021fedb21855cdb3cc57762eac5